interpretation/ww_head.py

```diff
@@ -185,20 +199,21 @@ def process(message, isGameMaster = False):
             members_to_add = check.users(message)
             if members_to_add == False:
                 return [Mailbox().respond("I am sorry! I couldn't find the user you were looking for!",True)]
+            #TODO: Re-enable this
```

Comment:
Probably should re-enable this before merging. I can't see how it's disabled though?

Reply:
During testing, I temporarily commented the next two lines. This very comment can be removed.

Reply:
Can anyone remove this comment?